primedev/squirrel/squirrel.h

```diff
@@ -447,7 +448,7 @@ inline VoidFunction SQMessageBufferPushArg(Vector3& arg) {
 // Vectors
 template <ScriptContext context>
 inline VoidFunction SQMessageBufferPushArg(SQObject* arg) {
-	return [arg]{ g_pSquirrel<context>->pushSQObject(g_pSquirrel<context>->m_pSQVM->sqvm, arg); };
+	return [arg]{ g_pSquirrel<context>->pushobject(g_pSquirrel<context>->m_pSQVM->sqvm, arg); };
```

Comment:
I'm stupid, can you ELI5 this change to me? ^^"

Reply:
Somebody failed their spelling bee and the compiler for some reason didnt figure it out until I tried to call the function, and THEN it started complaining about `pushSQObject` not existing (because it is supposed to be `pushobject` instead.)

Basically, compiler is dumb.

Reply:
Copy that, fast tracking that fix o7